css/style.css

```diff
@@ -429,6 +479,105 @@ ul {
 
 /* 1st subpage - About Us */
 
+.about-section {
+padding-top: 5px;
+display: grid;
+grid-template-columns: 1fr 1fr;
+}
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-left: 60px;
+}
+
+.image__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-title {
+  font-family: "Roboto-Black", sans-serif;
+  font-size: 3rem;
+  color: var(--text-color-dark);
+  margin-bottom: 5px;
+  text-align: center;
+} 
+ 
+.about-text {
+  font-family: "Roboto-Regular", sans-serif;
+  font-size: 0.75rem;
+  color: var(--text-color-dark);
+  text-align: justify;
+  margin-right: 5%;
+  padding-top: 1.5%;
+  line-height: 1.5;
+  font-size: 1.5rem;;
+}
+
+.about-img {
+  border-radius: 20px;
+  max-width: 100%;
+}
+
+ 
+/*Mobile view*/
+@media (max-width: 768px) {
+  .about-section {
+    grid-template-columns: 1fr;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .about-container {
+    margin: 0;
+    padding: 10px;
+  }
+
+  .about-title {
+    font-size: 1.5rem;
+  }
+
+  .about-text {
+    font-size: 1rem;
+  }
+
+  .about-img {
+    border-radius: 20px 20px 0 0 ;
+  }
+
+ 
+}
+/*Tablet view*/
+@media (min-width: 768px) {
+
+  .about-section {
+    margin: 0;
+  }
+
+  .about-container {
+    padding: 0 20px;
+    margin: 0;
+  }
+  .about-title {
+    font-size: 2.5rem;
+  }
+
+  .about-text {
+    font-size: 1.5rem;
+  }
+  .about-img {
+    border-radius: 10px;
+  }
+
+}
+ 
+
```

Comment:
empty spaces

Reply:
The empty spaces is a just a way to leave some space between this section and new theme / section etc